tensorbay/opendataset/VOC2012Detection/catalog.json

```diff
@@ -0,0 +1,84 @@
+{
+    "BOX2D": {
+        "categories": [
```

Comment:
If the order of categories is not important, better to sort the categories in order.
Same with attributes.